content/en/_index.html

```diff
@@ -28,7 +28,7 @@
 {{% /blocks/feature %}}
 
 {{% blocks/feature image="suitcase" %}}
-#### Run Anywhere
+#### Run K8s Anywhere
```

Comment:
I think the original meaning of this was "Kubernetes Runs Anywhere", not "Run Kubernetes Anywhere".

Also I'm not sure we want to use "k8s" here.

Reply:
I don't think there's a meaningful enough difference between the two:

i.e.:

> Run K8s Anywhere
> Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.

vs.

> K8s Runs Anywhere
> Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.

.... Both of those mean virtually the same thing from a language standpoint. One is (You can) Run Kubernetes Anywhere (because it's open source). The other is Kubernetes Runs Anywhere (because it's open source).

The boss' instructions on this one were to use "K8s" on one of the subheaders on the homepage, for legal copyright reasons. If we don't like the phrasing on this header, I'd suggest changing the 1st one instead 

from
> Planet Scale
to 
> K8s is Planet Scale

Thoughts, @tengqm?

Reply:
"K8s is Planet Scale" sounds good.